src/data_template_generators.js

```diff
@@ -48,7 +48,9 @@ const generateDataTemplate = (courseData, pathLookup) => ({
   open_learning_library_versions: helpers.getOpenLearningLibraryVersions(
     courseData["open_learning_library_related"]
   ),
-  legacy_uid: helpers.addDashesToUid(courseData["uid"])
+  legacy_uid:      helpers.addDashesToUid(courseData["uid"]),
+  highlights_text: courseData["highlights_text"] || "",
```

Comment:
Can we also make these changes to `generateLegacyDataTemplate`?  For some history on this:

Initially the `import_ocw_course_sites` management command in `ocw-studio` parsed the same `course.json` data template that's used by `ocw-hugo-themes` to actually render courses, but we had to change this Once we started handling thils like instructors by UUID reference.  Since `ocw-to-hugo` is the original source of truth for courses in studio, the full instructor data needed to be written out for the purposes of import, but we wanted the `course.json` files to match the schema that `ocw-studio` uses exactly so we didn't have to do any special casing in the theme.  Therefore, we ended up with two versions of the course data template, one just called `course.json` and another called `course_legacy.json`.  The `course_legacy.json` file should mirror `course.json`, except for data like instructors and the course images all of the properties are included, not just the UUID.  I hope this makes sense, p/ease ask if you need clarification on anything.

Reply:
@gumaerc 
I believe you already added these fields in the following PR
https://github.com/mitodl/ocw-to-hugo/pull/457/files#diff-19c6012e03f2a690ce54a5eb8e83d6fd6635c8674f93f08fbe7da54e573f05bc

Reply:
No problem.